zproject/settings.py

```diff
@@ -892,6 +899,10 @@ def zulip_path(path: str) -> str:
             'handlers': ['file', 'errors_file'],
             'propagate': False,
         },
+        'zulip.slow_queries': {
+            'level': 'DEBUG',
```

Comment:
I think `INFO` is more appropriate here?

Reply:
Depends on whether we end up adding some `.debug` calls with that logger, but for the current usage I guess `INFO` suits best.